apps/prairielearn/src/lib/group-update.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
     let params = { assessment_id };
     sqldb.queryOneRow(sql.select_assessment_info, params, function (err, result) {
       if (ERR(err, callback)) return;
-      const assessmentLabel = result.rows[0].assessmentLabel;
+      const assessmentLabel = result.rows[0].assessment_label;
```

Comment:
It seems to me that this column name is implicitly derived from the expression in:
https://github.com/PrairieLearn/PrairieLearn/blob/2193f2cc25a80282fabe0e54bb8566addc1ebb22/apps/prairielearn/src/lib/group-update.sql#L3

I think it would be clearer if that column has an explicit name.

Reply:
Added in 9adaf39!